common/dependabot-common.gemspec

```diff
@@ -20,8 +20,8 @@ Gem::Specification.new do |spec|
   }
 
   spec.version = Dependabot::VERSION
-  spec.required_ruby_version = ">= 3.1.0"
-  spec.required_rubygems_version = ">= 3.3.7"
+  spec.required_ruby_version = ">= 3.2.0"
+  spec.required_rubygems_version = ">= 3.3.7" # TODO: bump this to whatever version is bundled with ruby 3.2
```

Comment:
Not sure of the fastest way to figure this out? Not seeing anything obvious in the CI logs when I grep them for "rubygem"